discord/client.py

```diff
@@ -197,6 +198,26 @@ class Client:
         To enable these events, this must be set to ``True``. Defaults to ``False``.
 
         .. versionadded:: 2.0
+    bucket_storage_cls: :class:`type`[:class:`.rate_limiting.BucketStorageProtocol`]
```

Comment:
Are we sure this type definition is displayed correctly?

Reply:
It *should*, it's worked in the past with other projects I've done and v3 so I think this'll be fine.